### ISSUE TITLE
Not working on ACF Pro v5

### ISSUE THREAD

**Issue by barnabaslabs** [tamagokun/acf-forms#12]
Hi, 
I was using this successfully on a localhost site I'm building. Then I purchased ACF Pro. 

Now on the Form edit screen, the select box won't populate with the field groups. 

I haven't been able to figure it out & I'd really appreciate your insight!
Thanks!
Jack

**Comment by tamagokun** [comment]
@pastorjack I don't believe i've ever tested this with v5. I'm assuming some things have changed from v4 -> v5 which is causing the breakage. I can look it over when I have a chance, but if you are familiar with ACF, feel free to dig in. It's a pretty small plugin.

**Comment by tamagokun** [comment]
Are you referring to the select box that is in the meta box? It would be in this template: https://github.com/tamagokun/acf-forms/blob/master/views/form_meta_box.php

**Comment by barnabaslabs** [comment]
Yes that's the one!
